test/integration/scheduled-test.js

```diff
@@ -0,0 +1,45 @@
+let sinon = require('sinon')
+let test = require('tape')
+let { join } = require('path')
+let { scheduled } = require('../../src')
+let rateSpy = require('../mock/normal/src/scheduled/rate-scheduled')
+let cronSpy = require('../mock/normal/src/scheduled/cron-scheduled')
+
+const timeoutPromise = time => {
+  return new Promise(resolve => setTimeout(resolve, time))
+}
+
```

Comment:
the test here is performing a side effect and checking that the effect was executed.

i thought of other options as well, like mocking `child_process.spawn` but i felt this was the most realistic one, that would allow for future refactoring while still testing the functionality in a rather accurate way.

Reply:
I'm totally down with this approach, and have been meaning to apply it to `@events` integration tests as well. Can we be sure that it only writes into a known `tmp` folder (say `test/mock/tmp`) that we destroy as part of the test teardown? In my experience this has been a must when dealing with fs mutation during integration tests.